deploy/runtime/defaults/MSLF IPH_Commercial.json

```diff
@@ -20079,7 +20079,7 @@
     ],
     "ur_has_unused_items": 0.0,
     "ur_is_default": 0.0,
-    "ur_metering_option": 0.0,
+    "ur_metering_option": 4.0,
```

Comment:
Great - Buy all / Sell all and allowing  hourly/subhourly buy rates
![image](https://github.com/user-attachments/assets/d8a358d2-313c-4d2b-9110-8886d66e5aca)